lib/cli/interactive-setup/deploy.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+const Serverless = require('../../Serverless');
+const chalk = require('chalk');
+const { confirm, isUserLoggedIn, doesServiceInstanceHaveLinkedProvider } = require('./utils');
+const _ = require('lodash');
+const overrideStdoutWrite = require('process-utils/override-stdout-write');
+const { getDashboardInteractUrl } = require('@serverless/dashboard-plugin/lib/dashboard');
+const AWS = require('aws-sdk');
+
+const printMessage = ({
+  serviceName,
+  hasBeenDeployed,
+  dashboardPlugin,
+  isConfiguredWithDashboard,
+}) => {
+  if (isConfiguredWithDashboard) {
+    if (hasBeenDeployed) {
+      process.stdout.write(
+        `\n${chalk.green('Your project is live and available in ')}${chalk.white.bold(
+          `./${serviceName}`
+        )}\n`
+      );
+      process.stdout.write(`\n  Run ${chalk.bold('serverless info')} in the project directory\n`);
+      process.stdout.write('    View your endpoints and services\n');
+      process.stdout.write(`\n  Open ${chalk.bold(getDashboardInteractUrl(dashboardPlugin))}\n`);
+      process.stdout.write('    Invoke your functions and view logs in the dashboard\n');
+      process.stdout.write(`\n  Run ${chalk.bold('serverless deploy')} in the project directory\n`);
+      process.stdout.write(
+        "    Redeploy your service after you've updated your service code or configuration\n\n"
+      );
```

Comment:
nit: Maintainance wise it might be nicer to construct it with one `process.stdout.write` but simply group messages in `[].join("\n")` (also we may resort to template string, still then indentation could get quirky)

Reply:
Good call :+1: